slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -250,38 +253,41 @@ async def receive_messages(self) -> None:
                     break
                 try:
                     message: WSMessage = await session.receive()
-                    if self.trace_enabled and self.logger.level <= logging.DEBUG:
-                        # The following logging prints every single received message except empty message data ones.
-                        type = WSMsgType(message.type)
-                        message_type = type.name if type is not None else message.type
-                        message_data = message.data
-                        if isinstance(message_data, bytes):
-                            message_data = message_data.decode("utf-8")
-                        if len(message_data) > 0:
-                            # To skip the empty message that Slack server-side often sends
-                            self.logger.debug(
-                                f"Received message "
-                                f"(type: {message_type}, "
-                                f"data: {message_data}, "
-                                f"extra: {message.extra}, "
-                                f"session: {session_id})"
+                    # just in case, checking if the value is not None
+                    if message is not None:
```

Comment:
According to the type hint provided by aiohttp, we can assume the data cannot be `None` but we should check if it's not `None`, just in case (the behavior can be changed on the aiohttp library side in the future). In this PR, I've moved this check to cover the debug logging parts as well.